### PR TITLE
Mobile Responsive

### LINKED ISSUE #272: Mobile Responsive issue
### Screenshot - 
Screenshot (147)
### I would like to resolve this. Please assign it to me

### PR DESCRIPTION
**Pull request by Yashank18**
## Files Changed
* index.html
* js/site.js

 ### Before - 
Screenshot (147)

### After -
 
Screenshot (156)

### CODE REVIEW

File: js/site.js [hunk 1]
```diff
@@ -159,3 +159,16 @@ $(document).ready(function() {
...
+// to resize the backer Image
+function screenChange(screenW) {
+  if (screenW.matches) { // If media query matches
+    $("#lastOne").attr("data","https://opencollective.com/opensourcedesign/tiers/backer.svg?avatarHeight=50&width=320"); 
+  } else {
+     $("#lastOne").attr("data","https://opencollective.com/opensourcedesign/tiers/backer.svg?avatarHeight=50&width=400");
+  }
+}
+
+var screenW = window.matchMedia("(max-width: 400px)")
```
Comment (RDIL) [inline js/site.js:172]:
```suggestion
var screenW = window.matchMedia("(max-width: 400px)");
```

File: js/site.js [hunk 2]
```diff
@@ -159,3 +159,16 @@ $(document).ready(function() {
...
+screenChange(screenW) // Call listener function at run time
+screenW.addListener(screenChange)
```
Comment (RDIL) [inline js/site.js:174]:
```suggestion
screenW.addListener(screenChange);
```

File: js/site.js [hunk 3]
```diff
@@ -159,3 +159,16 @@ $(document).ready(function() {
...
+function screenChange(screenW) {
+  if (screenW.matches) { // If media query matches
+    $("#lastOne").attr("data","https://opencollective.com/opensourcedesign/tiers/backer.svg?avatarHeight=50&width=320"); 
+  } else {
+     $("#lastOne").attr("data","https://opencollective.com/opensourcedesign/tiers/backer.svg?avatarHeight=50&width=400");
+  }
+}
+
+var screenW = window.matchMedia("(max-width: 400px)")
+screenChange(screenW) // Call listener function at run time
```
Comment (RDIL) [inline js/site.js:173]:
```suggestion
screenChange(screenW); // Call listener function at run time
```

File: js/site.js [hunk 4]
```diff
@@ -159,3 +159,16 @@ $(document).ready(function() {
     }
   });
 });
+
+// to resize the backer Image
+function screenChange(screenW) {
+  if (screenW.matches) { // If media query matches
+    $("#lastOne").attr("data","https://opencollective.com/opensourcedesign/tiers/backer.svg?avatarHeight=50&width=320"); 
+  } else {
+     $("#lastOne").attr("data","https://opencollective.com/opensourcedesign/tiers/backer.svg?avatarHeight=50&width=400");
```
Comment (RDIL) [inline js/site.js:168]:
```suggestion
    $("#lastOne").attr("data","https://opencollective.com/opensourcedesign/tiers/backer.svg?avatarHeight=50&width=400");
```

**Comment by Yashank18** [comment]
@RDIL  can you please check now. I have done the changes

**Comment by jancborchardt** [comment]
By the way @Yashank18 since you are a member of the organization now, you can also work on branches of this repository. That makes it much easier for collaboration. :)

**Comment by Yashank18** [comment]
Okay, @jancborchardt I will surely take care of it from next time :)

**Pull request merged**